app/src/test/java/nerd/tuxmobil/fahrplan/congress/models/SessionTest.kt

```diff
@@ -120,7 +121,7 @@ class SessionTest {
     @Test
     fun `copy creates a new instance`() {
```

Comment:
Now that `Session` is a data class, I'd remove this test. If you keep it, you should change to `assertThat(session2)…`

Reply:
:heavy_check_mark: Removed.